runtime/moonbeam/src/precompiles.rs

```diff
@@ -177,3 +183,35 @@ where
 fn hash(a: u64) -> H160 {
 	H160::from_low_u64_be(a)
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
We usually test this in the runtime integration tests, not sure if we want to strat creating the precedent of testing here. Although I guess adding more tests should not hurt.

I usually implement things like https://github.com/PureStake/moonbeam/blob/80246854f3e69b583223cbe2c2634eb61279877e/runtime/moonbase/tests/integration_test.rs#L685, although it seems it has never been done for democracy

Reply:
Yea I was unsure about it as well, so did the minimum in order to gain additional discussion points.
Will hold on this until others can give their 2 cents.

Reply:
We already test precompiles existence (and non existence) here:
https://github.com/PureStake/moonbeam/blob/master/runtime/moonbeam/tests/integration_test.rs#L1919